dptb/tests/test_occupy.py

```
@@ -0,0 +1,847 @@
+        The C dimer has an outlier eigenvalue at +76.9 eV (sigma* anti-bonding),
+        which created multiple local minima in the error function. The brent
+        optimizer would converge to error=6 (14 electrons) instead of error=0
+        (8 electrons).
+
+        Band structure:
+        - Bands 1-4: -18.6 to -9.3 eV (bonding orbitals)
+        - Band 5: -7.6 eV
+        - Bands 6-7: +2.3 eV (pi* anti-bonding)
+        - Band 8: +76.9 eV (sigma* anti-bonding - OUTLIER)
+
+        For 8 electrons (2 C atoms × 4 valence electrons), we need 4 filled bands.
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)